v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.js

```diff
@@ -90,6 +91,27 @@ router.get('/', function(req, res, next) {
                 callback(null);
             });
         },
+        // We can probably combine the previous function with the following one, and eliminate sql.get_submission
+        function(callback) {
+            res.locals.showAllSubmissions = false;
+            submissionFilePath = path.join(res.locals.course.path, "questions", res.locals.question.directory, "submission.html");
```

Comment:
I think it's fine to display the fact that there were submissions even though we can't display the submission itself. Otherwise it would be confusing that some questions would appear to not have submissions and others would have them, which might lead people to think that the invisible submissions were not actually saved.

However, I think we should modify the question code (for `Calculation` questions now, at least) to render `Submission not shown` or similar for questions that don't have a `submission.html`.